<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticsSetupPage</class>
 <widget class="QWizardPage" name="StatisticsSetupPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,1,0">
      <item row="0" column="1">
       <widget class="QToolButton" name="addFileButton">
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="removeFileButton">
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::ListRemove"/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Output directory:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="outputDirOutput">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="changeOutputDirButtonButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1" rowspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>229</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" rowspan="3">
       <widget class="QListView" name="listView"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
